src/theory/quantifiers/instantiate.cpp

```diff
@@ -591,19 +591,13 @@ bool Instantiate::recordInstantiationInternal(Node q, std::vector<Node>& terms)
   {
     Trace("inst-add-debug")
         << "Adding into context-dependent inst trie" << std::endl;
-    CDInstMatchTrie* imt;
-    std::map<Node, CDInstMatchTrie*>::iterator it = d_c_inst_match_trie.find(q);
-    if (it != d_c_inst_match_trie.end())
-    {
-      imt = it->second;
-    }
-    else
+    const auto [it, was_inserted] = d_c_inst_match_trie.insert({q, nullptr});
```

Comment:
Nit: We generally use camelCase for variable names.

Reply:
Right, done